apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/AppConfiguration.java

```diff
@@ -106,16 +106,4 @@ public static AppConfiguration loadConfig(ConfigLoader configLoader) {
         sqsClient,
         sqsQueueName);
   }
-
-  /**
-   * Creates a {@link SqsClient} instance using settings from the provided {@link AwsClientConfig}.
-   *
-   * @param awsClientConfig used to configure AWS services
-   * @return the {@link SqsClient}
-   */
-  static SqsClient createSqsClient(AwsClientConfig awsClientConfig) {
-    final var clientBuilder = SqsClient.builder();
-    awsClientConfig.configureAwsService(clientBuilder);
-    return clientBuilder.build();
-  }
```

Comment:
This has been moved into `SqsTestUtils` so that it can also be used in the pipeline.